### DISCUSSION TITLE
Password Rest Email

### DISCUSSION THREAD

**Discussion by imknight** [supabase/supabase#11643] [Questions]
I have build a static page to update password once the reset email sent. The problem is now the link attach in the reset email can only click once. If you click the link, close it without update the password, the next time you try to click again, the link no longer work.

This is causing some problem for my client who use Microsoft outlook, it seem like there are some proxy scan through the link, the link that the user receive through their email is always expired ( I assume the link been click when the proxy scanning through the email content), any suggestion what can be done?

**Comment by GaryAustin1** [comment]
You could send a OTP code with {{.Token}} in the email template with an explanation to use it if the other does not work (or just use the OTP method).   Then have a screen where they enter the OTP code, you verify it and they get access to your password reset page.

**Reply by imknight** [reply]
don't think it work, the problem is as long as the email go through the client email system, the link will be trigger and no longer work.

**Reply by GaryAustin1** [reply]
Ah, Yeah that makes sense.  So then you are {{.Token}} only and not putting the URL in at all.  I do that (OTP only) but I can't have links for a homescreen PWA.
